package.json

```diff
@@ -19,6 +19,8 @@
     "docker:mssql:start": "docker run -e 'ACCEPT_EULA=Y' -e 'SA_PASSWORD=yourStrong(!)Password' -p 1433:1433 -d microsoft/mssql-server-linux:latest",
     "docker:oracle:build": "docker build test/docker/oracle -t falcon-test-oracle --no-cache",
     "docker:oracle:start": "docker run --rm -ti -p 1521:1521 falcon-test-oracle",
+    "docker:clickhouse:build": "docker build test/docker/clickhouse -t falcon-test-clickhouse --no-cache",
```

Comment:
Unless there is a good reason for keeping `--no-cache`, we should remove it. It would just slow things for developers.